source/extensions/clusters/redis/redis_cluster.cc

```diff
@@ -228,6 +232,9 @@ RedisCluster::RedisDiscoverySession::~RedisDiscoverySession() {
     current_request_->cancel();
     current_request_ = nullptr;
   }
+  if (resolve_timer_) {
+    resolve_timer_->disableTimer();
+  }
```

Comment:
Is this needed? If so comment?

Reply:
I'll add a comment.  Sorry I missed this one.